src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,21 @@
 import React from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const { name, email } = user;
+
+  return (
+    <p>
+      {' Posted by  '}
```

Comment:
Consider removing the extra space before 'Posted by' to keep the text consistent.